externs/olx.js

```diff
@@ -2734,6 +2734,17 @@ olx.interaction.DrawOptions.prototype.wrapX;
 
 
 /**
+ * @typedef {{extent: (ol.Extent|undefined),
+ *     extentProjection: (ol.proj.ProjectionLike|undefined),
+ *     bboxStyle: (ol.style.Style|Array.<ol.style.Style>|ol.style.StyleFunction|undefined),
+ *     pointerStyle: (ol.style.Style|Array.<ol.style.Style>|ol.style.StyleFunction|undefined),
+ *     wrapX: (boolean|undefined)}}
+ * @api
+ */
+olx.interaction.ExtentOptions;
```

Comment:
In addition to this typedef, you'll need to define each option as `@type` below (see other options in `externs/olx.js`).